app/models/comfy/cms/page.rb

```diff
@@ -97,7 +97,7 @@ def published_at
     elsif scheduled?
       scheduled_on
     elsif ever_been_published?
-      revisions.detect { |revision| revision.data[:event].include?('published') }.created_at
+      revisions.detect { |revision| Array(revision.data[:event]).include?('published') }.created_at
```

Comment:
Yeah, but revision.data[:event] returns a String (or apparently nil), at least according to the specs. This is turning it into an array of strings, containing one string, or an empty array.

![voila_capture 2015-05-18_12-15-22_pm](https://cloud.githubusercontent.com/assets/2986/7679738/904d5014-fd57-11e4-91ce-1d8c8e00b9c5.png)